leaflet_export.js

```diff
@@ -260,6 +260,32 @@
 
       this.downloadExport = function (options) {
         options = options || {};
+        
+        if (Ember.isArray(options)) {
```

Comment:
it is not EmberJS addon, but Leaflet one, so Ember functions will not be presented there always. Use ES5 Array.isArray method or just check options.length